### DISCUSSION TITLE
reference to column position without --no-header option

### DISCUSSION THREAD

**Discussion by mirkoperillo** [mithrandie/csvq#55] [Q&A] [answered]
Hi,
is it possible to reference to column position ( c1,c2 and so on) without using the --no-header name ?
Sometime the header names could be very verbose and so not comfortable to use in query.

This is a sample of verbose header names
https://www.istat.it/storage/codici-unita-amministrative/Elenco-comuni-italiani.csv

**Comment by mithrandie** [comment, marked as answer]
You can specify a column by using the table name and column number starting with 1 in the table.

```
$ cat sample.csv
column1,column2
1,a
2,b

$ cat sample2.csv
column3,column4
1,c
2,d

$ csvq 'SELECT t1.1, t1.2, t2.2 FROM `sample.csv` t1 INNER JOIN `sample2.csv` t2 ON t1.1 = t2.1'
+------+------+------+
| t1.1 | t1.2 | t2.2 |
+------+------+------+
| 1    | a    | c    |
| 2    | b    | d    |
+------+------+------+
```

**Reply by mirkoperillo** [reply]
Wonderful, exactly the result I want

Thank you a lot for your suggestion
